plugins/codex-memory/skills/soul-status/SKILL.md

```
@@ -0,0 +1,55 @@
+   for b in demarkus-server demarkus-mcp demarkus-token; do
+     printf '%s ' "$b"; ~/.demarkus/bin/$b --version 2>/dev/null || echo "(missing or too old for --version)"
+   done
+   ```
+
+   Note which binaries are present and what versions they report. A binary that
+   prints "(missing or too old for --version)" predates the `--version` flag and
+   will be upgraded on the next session start.
+
+3. **Check the server process.** Read `SOUL_DIR` and `PORT` from the config. Run:
+
+   ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)